src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -53,6 +53,13 @@ class PullRequestIssue:
 
 RATE_LIMITED_MESSAGE = "API rate limit exceeded"
 
+OPEN_PR_METRIC_BASE = "github_open_pr_comment.{key}"
+
+# Caps the number of files that can be modified in a PR to leave a comment
+OPEN_PR_MAX_FILES_CHANGED = 10
```

Comment:
Would it be better to lower this something like 5-7? 10 files feels like a large amount, and looking at our own PRs the vast majority of them are all below 10 files.

Reply:
Does this mean that this will only comment on up to the first 10 files?

Or does this mean that it will only comment on PRs that have <10 files changed?

Reply:
@nhsiehgit only comment on PRs with < 10 files changed AND < 500 lines modified